Backend/app.py

```diff
@@ -52,67 +54,99 @@ def getDatabaseClient():
   on if we are in Dev or Prod
   '''
   if prod == 'True':
+    logger.info("Using AWS Prod Database")
     return boto3.resource('dynamodb')
   else:
-    print("Returning the Dev resource")
+    logger.info("Using Local Dev Database")
     return boto3.resource('dynamodb', endpoint_url='http://localhost:8000')
 
 @auth.verify_password
 def verify_password(email_or_token, password):
+  logger.info(str(request))
   dynamoDBInstance = getDatabaseClient()
-  return MentiiAuth.verify_password(email_or_token, password, dynamoDBInstance, appSecret)
+  response = MentiiAuth.verify_password(email_or_token, password, dynamoDBInstance, appSecret)
+  logger.info(str(response))
```

Comment:
I agree that logging True/False is basically useless but I disagree that more information should be logged. The only extra information I could see someone adding here is whether the email wasnt found or if the supplied password doesnt match the password for the supplied email. From a security perspective you shouldnt really disclose either. Im not sure that there would be value in logging a user entering the wrong credentials anyway. My vote would be just remove this log altogether